python/ray/tune/tuner.py

```diff
@@ -167,6 +167,14 @@ def restore(
         resume_unfinished: bool = True,
         resume_errored: bool = False,
         restart_errored: bool = False,
+        overwrite_trainable: Optional[
+            Union[
+                str,
+                Callable,
+                Type[Trainable],
+                "BaseTrainer",
+            ]
```

Comment:
Unrelated to this PR, but we should alias this 😄

Reply:
+1 can we just define this at the top of the file

```python
TrainableType = Optional[Union[str, Callable, Type[Trainable], "BaseTrainer"]
```